workbench/src/main/main.js

```diff
@@ -90,7 +90,7 @@ export const createWindow = async () => {
   setupRendererLogger();
   await getFlaskIsReady();
 
-  const devModeArg = ELECTRON_DEV_MODE ? '--devMode' : '';
+  const devModeArg = ELECTRON_DEV_MODE ? '--devmode' : '';
```

Comment:
this was silently getting lowercased anyway, which was messing with our args parsing in `preload/api.js`